qtOgitor/CMakeLists.txt

```diff
@@ -143,6 +144,21 @@ else(NOT Qt5Widgets_FOUND)
         ./include/uiterrainsquare.hxx
         ./include/userdefined.hxx
     )
+    
+    SET(QTOGITOR_TS_FILES

```

Comment:
..put and `if(Qt5LinguistTools_FOUND)` around this part. As the linguist tools are an optional package in universe on Ubuntu and one can easily live without translations.